### PR TITLE
[OOT] 4 new sometimes hints

### PR DESCRIPTION
**Pull request by Celestial-Panda**
### Done
- Water Temple Central Pillar chest
- Water Temple BK chest
- Shadow Temple Flaming Pot

### Still need work
- Water Temple River chest + GS if tokensanity is on

### CODE REVIEW

File: src/common/text.c [hunk 1]
```diff
@@ -359,7 +359,7 @@ const RegionName kRegionNamesOot[] = {
     { "in",         "the " TEXT_COLOR_YELLOW "Sacred Realm" },
     { "inside",     "the " TEXT_COLOR_GREEN "Deku Tree" },
     { "inside",     TEXT_COLOR_RED "Dodongo's Cavern" },
-    { "inside",     TEXT_COLOR_BLUE "Jabu-Jabu" },
+    { "inside",     TEXT_COLOR_BLUE "Jabu Jabu's Belly" },
```
Comment (Nax) [inline src/common/text.c:362]: The game has the hyphen there

File: src/common/text.c [hunk 2]
```diff
@@ -368,20 +368,20 @@ const RegionName kRegionNamesOot[] = {
...
+    { "in",         TEXT_COLOR_YELLOW "Hyrule Castle" },
     { "in",         TEXT_COLOR_RED "Ganon's Castle Exterior" },
     { "in",         "the " TEXT_COLOR_GREEN "Lost Woods" },
-    { "in",         "the " TEXT_COLOR_GREEN "Sacred Meadow" },
+    { "in",         "the " TEXT_COLOR_GREEN "Sacred Forest Meadow" },
     { "in",         TEXT_COLOR_RED "Kakariko" },
-    { "inside",     TEXT_COLOR_PINK "Kakariko's Graveyard" },
-    { "on",         TEXT_COLOR_RED "Death Mountain's Trail" },
-    { "in",         TEXT_COLOR_RED "Death Mountain's Crater" },
+    { "inside",     TEXT_COLOR_PINK "Graveyard" },
```
Comment (Nax) [inline src/common/text.c:382]: There are two graveyards (with Ikana), so it's better to name both

File: src/common/text.c (hunk 1 above)
Comment (Celestial-Panda) [reply src/common/text.c:362]: Wouldn't be on par with OOTR hints, but I suppose - [main OOTR hints code](https://github.com/TestRunnerSRL/OoT-Randomizer/blob/Dev/Hints.py#L330)

File: src/common/text.c (hunk 2 above)
Comment (Celestial-Panda) [reply src/common/text.c:382]: I suppose that's fair, although MM's Graveyard is already demarcated by "Ikana" in front of it. But I'll just revert that back, not a big deal, just weird

**Comment by Celestial-Panda** [comment]
Alright, now that I'm on winter break, is there a way with the current hint system to have Conditional Hints? I.e if OOT tokensanity is either set to all or dungeons, the river hint has both the skulltula and the chest, but if tokensanity is off, only the chest gets hinted?

**Pull request closed without merging**